data/vijaysheru.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Vijay Sheru",
+  "type": "personal",
+  "bio": "Mechanical Enginner turned Software Developer-Arts,Design,Artificial Intelligence,Code,Cryto,Cloud"
```

There is a missing comma `,` leading json check failure. 

```suggestion
  "bio": "Mechanical Enginner turned Software Developer-Arts,Design,Artificial Intelligence,Code,Cryto,Cloud",
```